src/cpp/linearstoppotential.h

```diff
@@ -77,7 +78,7 @@ class LinearStop {
 
   std::string logging_filename_;
   std::unique_ptr<RAIIFile> logging_file_{nullptr};
-  bool stop_not_called_{true};
+  bool stop_called{false};
```

Comment:
Let's keep consistent with the value referenced in line 61 and with the GROMACS naming conventions for member variables (trailing underscore).
```suggestion
  bool stop_called_{false};
```

Reply:
We also need to update the source file where this value is checked and used, changing it from a local variable to the member variable.

Reply:
Changed `stop_called` to `stop_called_` in second commit!